src/api/decoding-scanned-codes.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+parent: API Reference
+title: Decoding Scanned Codes
+permalink: /api/decoding-scanned-codes
+---
+
+# Decoding Scanned Codes
+
+A scanned code can be a either QR code or a barcode that a merchant or a patron scans.
+The scanned code can be decoded to check the provider of the code.
+
+## Contents
+{:.no_toc .text-delta}
+
+* TOC
+{:toc}
+
+## Models
+
+### Scanned Code
+
+{% h4 Mandatory Fields %}
+
+|    Field    |        Type        |                           Description                            |
+| :---------- | :----------------- | :--------------------------------------------------------------- |
+| code        | String             | The code that has been scanned.                                  |
+| scannedBy   | String             | The entity that scanned the code. Can be `merchant` or `patron`. |
+| provider    | String             | The provider for the scanned code.                               |
```

Comment:
Scanned code is the name of this page. Provider is the name of this field. "The provider for the scanned code" says nothing about the field. Please reword to give context, what are the contents of this field and how does it change behavior of the response?

Reply:
I've changed this now. I'm unsure what you mean by "how does it change behaviour of the response". Could you elaborate/give an example of what you mean?

Reply:
Hmm, reading this again I realize that this isn't something you push in your POST body. You can ignore this part.